src/journey.js

```diff
@@ -1,5 +1,6 @@
 import { all, assoc, equals, findLast, pipe, reject } from "ramda";
 import { v4 } from "uuid";
+var seedrandom = require('seedrandom');
```

Comment:
This should be a `const`. Also, can we use the `import` syntax here, instead of the `require` ?

Reply:
I was actually trying to do this earlier, figured it out now though: Changed this in: 66fbdb0